add: readme and contributing guide (#72)
devpt-org/discord-bot: CONTRIBUTING.md

# Contribuição

> Antes de mais, agradecimentos por considerares participar neste projeto.

Reviewer: ```suggestion
Antes de mais, obrigado por considerares participar neste projeto.
```

> Neste documento podes encontrar uma série de linhas para contribuir neste projeto. Estas instruções não são para ser consideradas regras rígidas mas guias, esperamos que cada pessoa use bom senso em cada participação.

Reviewer: ```suggestion
Neste documento encontram-se instruções detalhadas para que possas contribuir em conformidade com a estrutura e organização do projecto. Estas instruções não são para ser consideradas regras rígidas mas sim uma orientação generalizada do que esperamos quando alguém contribui, utiliza o bom senso quando estiveres a contribuir para o projecto.
```

#### Tabela de conteúdo

1. [Código de conduta](#código-de-conduta)
2. [Arquitetura](#arquitetura)
3. [Linguagem e versões](#linguagem-e-versões)
4. [Guia de estilos](#guia-de-estilos)
   - [Código](#código)
   - [Issues](#issues)
   - [Mensagens de commit](#mensagens-de-commit)

## Código de conduta

> Para participar neste projeto, esperamos que cada indivíduo siga as seguintes regras:

Reviewer: ```suggestion
Ao participares neste projecto, esperamos que tenhas em consideração as seguintes regras:
```

- Respeito pelo próximo
- Uso de linguagem inclusiva e acolhedora
- Aceitação de crítica construtiva
- Foco no que é melhor para a comunidade

## Arquitetura

- Este projeto segue, na sua grande maioria, o estilo de _Domain-Driven Design_ (DDD). Para mais informação consulta o nosso documento sobre [Arquitetura](ARCHITECTURE)

## Linguagem e versões

> - A linguagem utilizada é Typescript. Para instalar as dependências precisas de ter acesso a Node e NPM. A versão de Node necessária pode ser consultada no [ReadMe](README).

Reviewer: ```suggestion
- A linguagem utilizada é Typescript, e tem Node e NPM como dependências principais. As versões necessárias podem ser consultadas no [ReadMe](README).
```
